apps/cookbook/src/app/examples/modal-example/modal-example-outlet.component.ts

```diff
@@ -93,7 +94,7 @@ export class ModalExampleOutletComponent {
 
   constructor(private modalController: ModalController) {}
 
-  navigateToModalRoute(path: string | string[]) {
-    this.modalController.navigateToModal(path);
+  navigateToModalRoute(path: string | string[], queryParams: Params | null) {
```

Comment:
Why not an optional param? 🤔 
```suggestion
  navigateToModalRoute(path: string | string[], queryParams?: Params) {
```